### DISCUSSION TITLE
VENLAFLAXINE PBPK Project

### DISCUSSION THREAD

**Discussion by harishkaushikbugworks** [Open-Systems-Pharmacology/Forum#821] [Questions & Problems] [answered]
Hello All,

If any one has Venlaflaxine PBPK Project , if you can share it will help 

https://faseb.onlinelibrary.wiley.com/doi/abs/10.1096/fasebj.2020.34.s1.03769?af=R

I saw this publication in literature, but no further details on the methodology, if you can help it will be useful

Regards
Harish

**Comment by StephanSchaller** [comment, marked as answer]
Dear Harish (@harishkaushikbugworks),

I believe this model is not available here, but maybe you can reach out to the author and ask if he is willing to share and upload it to the OSP GitHub Site?

Thanks,
Stephan
